src/ConnectedMode.UnitTests/Migration/ObsoleteConnectedModeSolutionBindingPathProviderTests.cs

```diff
@@ -19,91 +19,60 @@
  */
 
 using System;
-using Microsoft.VisualStudio.Shell.Interop;
 using SonarLint.VisualStudio.ConnectedMode.Migration;
+using SonarLint.VisualStudio.Core;
 
 namespace SonarLint.VisualStudio.ConnectedMode.UnitTests.Migration
 {
     [TestClass]
     public class ObsoleteConnectedModeSolutionBindingPathProviderTests
     {
-        private Mock<IVsSolution> solution;
```

Comment:
Miscellaneous: changed the existing tests to remove the `TestInitialize` in favour of a `CreateTestSubject` method.